analyses/cell-type-consensus/references/panglao-cell-type-ontologies.tsv

```diff
@@ -85,27 +85,27 @@ CL:0000893	thymocyte	Thymocytes
 CL:4023169	trigeminal neuron	Trigeminal neurons
 CL:0000351	trophoblast cell	Trophoblast cells
 CL:0000731	urothelial cell	Urothelial cells
-NA	NA	Adipocyte progenitor cells
-NA	NA	Airway epithelial cells
-NA	NA	Airway goblet cells
-NA	NA	Airway smooth muscle cells
-NA	NA	Alpha cells
-NA	NA	Anterior pituitary gland cells
-NA	NA	B cells memory
-NA	NA	B cells naive
-NA	NA	Bergmann glia
-NA	NA	Beta cells
-NA	NA	Cajal-Retzius cells
-NA	NA	Cardiac stem and precursor cells
-NA	NA	Cardiomyocytes
-NA	NA	Choroid plexus cells
-NA	NA	Clara cells
-NA	NA	Crypt cells
-NA	NA	Delta cells
-NA	NA	Distal tubule cells
-NA	NA	Ductal cells
-NA	NA	Endothelial cells (aorta)
-NA	NA	Endothelial cells (blood brain barrier)
+CL:0002334	preadipocyte	Adipocyte progenitor cells
+CL:0002368	respiratory epithelial cell	Airway epithelial cells
+CL:0002370	respiratory goblet cell	Airway goblet cells
+CL:0019019	tracheobronchial smooth muscle cell	Airway smooth muscle cells
+CL:0000171	pancreatic A cell	Alpha cells
+CL:0000637	chromophil cell of anterior pituitary gland	Anterior pituitary gland cells
+CL:0000787	memory B cell	B cells memory
+CL:0000788	naive B cell	B cells naive
+CL:0000644	Bergmann glial cell	Bergmann glia
+CL:0000169	type B pancreatic cell	Beta cells
+CL:0000695	Cajal-Retzius cell	Cajal-Retzius cells
+CL:0002664	cardioblast	Cardiac stem and precursor cells
+CL:0000746	cardiac muscle cell	Cardiomyocytes
+CL:0000706	choroid plexus epithelial cell	Choroid plexus cells
```

Comment:
Do we know they mean epithelial cells? Looks like fibroblasts, macrophages, stromal cells are possible

Reply:
I chose this term because the broad synonym was `choroid plexus cell`, while the macrophage [term](https://www.ebi.ac.uk/ols4/ontologies/cl/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCL_4042004) doesn't list that as a synonym. I checked the other terms and also didn't see that listed as a synonym.